src/tunnelManager.c

```diff
@@ -100,6 +101,15 @@ ioTunnel *addIoPlugin(const char *libname)
 
 	handle = dlopen( libname, RTLD_NOW);
 		
+	if(handle == NULL) {
+		fullpath = malloc(strlen(TUNNELLIBDIR) + strlen(libname) + 2);
```

Comment:
It is defined by the

AC_DEFINE_UNQUOTED(TUNNELLIBDIR, ...

in configure.ac. This propagates to src/debug_level.h which contains

#define TUNNELLIBDIR ...

src/tunnelManager.c has:

#include "debug_level.h"